### ISSUE TITLE
385 audience characteristics

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#174]
https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/3XX.csv

**Comment by CECSpecialistI** [comment]
@GordonDunsire , if a code value is given for MARC 385 with a $2 for the source, and there is no $a/$0/$1, does RDA view this code as a structured value or an identifier?

**Comment by GordonDunsire** [comment]
@CECSpecialistI: I think $2 for source code in any tag should be treated as an identifier. In general, RDA treats codes as identifiers. An expansion or resolution of the code is treated as a structured description because it is a controlled term.
A general "mapping" from typical LC controlled vocabularies to RDA is:
Term/concept = structured description
Code/notation = identifier
IRI = IRI

**Comment by CECSpecialistI** [comment]
Thanks @GordonDunsire !

**Comment by gerontakos** [comment]
This is pertinent to what I worked on yesterday regarding $2 so let me ask a question.
Here's a 385:  385 ## $a Painters $2 lcsh
It's clear enough that "Painters" is an RDA structured value  and "lcsh" is an identifier for the source of that structured value. Still, how do we map it? I think 3 things are on-the-table at present: (1) local property creation extending RDA, (2) statement reification that allows description of data provenance, (3) use a custom datatype for the source. As I said elsewhere, I'm favoring (3) at present:
<rdf:Description rdf:about="https://open-na.hosted.exlibrisgroup.com/alma/01ALLIANCE_UW/bf/entity/123456789#Expression">
<rdawe:P20322 rdf:datatype="[http://id.loc.gov/authorities/subjects](http://id.loc.gov/authorities/subjects)"> Painters </rdawe:P20322></rdf:Description>

**Comment by CECSpecialistI** [comment]
I'm favoring (3) as well...it seems like the most efficient way to facilitate reconciling these against LCSH (or other sources that are used here such as lcdgt and wikidata) later.

**Comment by gerontakos** [comment]
Agreed. Easy to query (SPARQL has the "datatype" function, for example).

**Comment by CECSpecialistI** [comment]
Hi @SitaKB , would you review this mapping?

**Comment by CECSpecialistI** [comment]
@CECSpecialistI will overhaul mapping spreadsheet, @cspayne will review and help @arajapu implement.

**Comment by CECSpecialistI** [comment]
I've overhauled the mapping spreadsheet today. Ready for @cspayne 's review
